### ISSUE TITLE
Point Cloud ghost pixels

### ISSUE THREAD

**Issue by greenbreakfast** [OnionIoT/tau-lidar-server#17]
### Python Library versions
> Run `pip list | grep TauLidar` in the command-line and paste the output. Include some details on your system as well

```
TauLidarCamera            0.0.1
TauLidarCommon            0.0.1
TauLidarServer            0.0.1.post2
```

Running on Mac

### Camera Info
> If possible, paste the output of `camera.info()` from Python (after camera has been successfully opened)

```
ToF camera opened successfully:
    model:      4.0
    firmware:   3.3
    uid:        69.549
    resolution: 160x60
    port:       /dev/cu.usbmodem00000000001A1
    IP address: 127.0.0.1
    URL:  http://127.0.0.1:8081
```

### Describe the bug

The point cloud visualization in the browser holds on to "ghost" pixels. These are pixels that had a depth measurement in a previous camera frame, but after the camera was moved there was no depth data for this x,y coordinate, resulting in the previous depth measurement staying in the point cloud and not being cleared.

This results in some strange looking frames

### Steps to Reproduce

1. Run the TauLidarServer python program
1. point the camera at a scene
1. move the camera to point at a scene where some parts of the depth map are an undefined distance
1. Observe point cloud has "ghost" pixels left over from the frames of scene in step 2 overlaid with the point cloud from the scene in step 3

### What is expected?

The point cloud should not include any points from previous frames. Each frame should be independent

### What is actually happening?

Parts of the point cloud that used to have a distance measurement and now do not stay in the point cloud.

Previous frame:
![Screen Shot 2020-11-27 at 4 19 46 PM](https://user-images.githubusercontent.com/6757320/100485007-88b5a780-30cc-11eb-8d56-d378a6345176.png)

Next frame with ghost pixels circled in green:
![Screen Shot 2020-11-27 at 4 19 57 PM](https://user-images.githubusercontent.com/6757320/100485020-90754c00-30cc-11eb-81b9-8fb10c14a1e4.png)

### Any additional comments?

See screen capture for an illustration of the issue at 0:07 and 0:12 
https://youtu.be/hVjc9lZJRXA

**Comment by mh1412** [comment]
Ghost pixels remains when updated frame is missing data on the corresponding coordinates. 
Issue can be resolved by reinitializing the point cloud scene every time when new frame updates. 

Fixed in 6102a79e08341e6a275764a4fbf2a5108aa4a7d5

**Comment by greenbreakfast** [comment]
released in [v0.0.3](https://github.com/OnionIoT/tau-lidar-server/releases/tag/v0.0.3)
